caffe2/core/macros.h.in

```diff
@@ -81,5 +80,4 @@ static_assert(
   {"USE_MKLDNN", "${CAFFE2_USE_MKLDNN}"}, \
   {"USE_NVTX", "${CAFFE2_USE_NVTX}"}, \
   {"USE_TRT", "${CAFFE2_USE_TRT}"}, \
-  {"DISABLE_NUMA", "${CAFFE2_DISABLE_NUMA}"}, \
```

Comment:
Wow! Is there really no where else that uses `CAFFE2_DISABLE_NUMA`? I'm impressed

Reply:
yeah that is true